hw2/03-count.js

```diff
@@ -1,3 +1,31 @@
-// Add your code here
+const input = document.querySelector("#userInput");
+input.addEventListener("keydown", handleKeyDown); // grab input on enter
 
-input.addEventListener('keydown', handleKeyDown);
+// Search and replace inputted word globally with highlighted word
+function handleKeyDown(event) {
+  const word = input.value.trim();
+
+  // Variable containing searchable content
+  const textBody = document.querySelector(".card-text");
+
+  const regex = new RegExp(word, "gi"); // g = global search
+
+  // Create a highlighted word to replace the match
+  const highlightedWord = document.createElement("span");
+  highlightedWord.textContent = word;
+  highlightedWord.style.backgroundColor = "yellow";
```

Comment:
Interesting use of span rather than mark! It appears that you were able to avoid the use of string templates because of this choice. I like it